src/main/java/nextstep/ladder/domain/Line.java

```diff
@@ -2,27 +2,62 @@
 
 import java.util.ArrayList;
 import java.util.List;
+import java.util.stream.Collectors;
 import java.util.stream.IntStream;
 
 public class Line {
-    private final List<Point> points;
+    private final List<Link> links;
 
-    public Line(int countOfPeople) {
-        points = new ArrayList<>();
+    public Line(boolean... links) {
+        this.links = IntStream.range(0, links.length)
+                .mapToObj(i -> new Link(links[i]))
+                .collect(Collectors.toList());
+    }
+
+    private Line(List<Link> links) {
+        this.links = links;
+    }
+
+    public static List<Line> createLines(int countOfLine, int countOfLink) {
+        List<Line> lines = new ArrayList<>();
+        IntStream.range(0, countOfLine)
+                .forEach(number -> lines.add(createLine(countOfLink)));
+
+        return lines;
```

Comment:
forEach가 아니라 map 으로 해도 되었을 것 같네요 ㅎ

```suggestion
        return IntStream.range(0, countOfLine)
                .map(number -> createLine(countOfLink))
                .toList();
```